web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -77,6 +77,22 @@ export function SelectResource({ onSelect }: SelectResourceProps) {
 
   const [search, setSearch] = useState('');
   const [resources, setResources] = useState<ResourceSpec[]>([]);
+  const userContext = ctx.storeUser.state;
+  const { acl, authType } = userContext;
```

Comment:
Nit, but it should be enough to do this, no?

```suggestion
  const { acl, authType } = ctx.storeUser.state;
```